specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/applicationGateway.json

```diff
@@ -485,9 +485,6 @@
               "$ref": "./network.json#/definitions/Error"
             }
           }
-        },
-        "x-ms-pageable": {
-          "nextLinkName": null
```

Comment:
List operations must usually be marked as `x-ms-pageable`, why is this being changed.

Reply:
@dsgouda I just added you in the email thread. Short version: the JSON returned by this operation is not compatible with a pageable and this cannot be generated as of today.